packages/@aws-cdk/aws-servicecatalog/rosetta/portfolio-product.ts-fixture

```diff
@@ -1,5 +1,6 @@
 // Fixture with packages imported, but nothing else
 import { Construct, Stack } from '@aws-cdk/core';
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
Can you instead add this import to those examples that need it? Will make them more self-contained.

Reply:
Unaddressed comment.

Reply:
whoops, I didn't see this comment.